Ports/halflife/package.sh

```diff
@@ -2,35 +2,46 @@
 port="halflife"
 version="1.0.0"
 useconfigure="true"
-depends=("SDL2")
-workdir="xash3d-fwgs-master"
+depends=("SDL2" "fontconfig" "freetype")
+workdir="."
 files="https://github.com/SerenityPorts/xash3d-fwgs/archive/master.tar.gz xash3d_engine.tar.gz
-https://github.com/SerenityPorts/hlsdk-xash3d/archive/master.tar.gz xash3d_hldll.tar.gz"
+https://github.com/SerenityPorts/hlsdk-xash3d/archive/master.tar.gz xash3d_hldll.tar.gz
+https://github.com/ccapitalK/vgui-dev/archive/master.tar.gz vgui-dev.tar.gz
```

Comment:
any reason not to use the repo you forked from? https://github.com/FWGS/vgui-dev Looks like your master is up to date w/theirs.

Reply:
Good point, it originally looked like I might need to make some changes to get it to build (and that might be true later as I try to get mouse support working), right now there isn't any reason to use it.